### PR TITLE
Feature/vector plus operator

### PR DESCRIPTION
**Pull request by marczaku**
// how to test

// what i've done

// blah blah

### CODE REVIEW

**Comment by marczaku** [comment]
Hello, comment!

**Review by mandelcohen — COMMENTED** [review]
Hello! I'm reviewing this.

File: Utility/Vector3.cs [hunk 1]
```diff
@@ -41,7 +41,7 @@ public static Vector3 Up
 
     public static Vector3 operator +(Vector3 a, Vector3 b)
     {
-        return new Vector3(a.x + b.x, a.y + b.z, a.z + b.z);
+        return new Vector3(a.x + b.x, a.y + b.y, a.z + b.z);
```
Comment (mandelcohen) [inline Utility/Vector3.cs:44]: Each arg in one line

Comment (mandelcohen) [reply Utility/Vector3.cs:44]: hello

**Review by mandelcohen — APPROVED** [review]
Looks good

**Pull request merged**